runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -94,6 +94,17 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
    bool isValueTypeClass(TR_OpaqueClassBlock *);
    bool isValueTypeClassFlattened(TR_OpaqueClassBlock *clazz);
    bool isValueBasedOrValueTypeClass(TR_OpaqueClassBlock *);
+   /** \brief
+    *	    Checks whether a class implements `IndentityObject`/`IdentityInterface`
+    *
+    *  \param clazz
+    *     The class that is to be checked
+    *
+    *  \return
+    *    `true` if the class implements `IndentityObject`/`IdentityInterface`;
+    *    `false` otherwise (that is, if the class is a value type)
```

Comment:
It can also be `false` for some abstract classes, interfaces and for `java/lang/Object`.

Reply:
I'll update it to
```
`false` otherwise (that is, if the class could be a value type, or some abstract class, interface, or java/lang/Object.
```

Reply:
Updated the comment in f7d8213